### ISSUE TITLE
error on install

### ISSUE THREAD

**Issue by Turqueso** [imagemin/imagemin-jpegoptim#8]
I'm getting this when i try `npm install --save imagemin-jpegoptim`

```
⚠ The `jpegoptim` binary doesn't seem to work correctly
⚠ jpegoptim pre-build test failed
ℹ compiling from source
```

**Comment by kevva** [comment]
Does it compile successfully then?

**Comment by philipbordallo** [comment]
Having the same issue and no, compiling from source doesn't seem to work either. 
```
⚠ The `jpegoptim` binary doesn't seem to work correctly
⚠ jpegoptim pre-build test failed
ℹ compiling from source
✖ Error: ./configure --prefix="/Users/philip/Code/gulp/images/node_modules/imagemin-jpegoptim/node_modules/jpegoptim-bin/vendor" --bindir="/Users/philip/Code/gulp/images/node_modules/imagemin-jpegoptim/node_modules/jpegoptim-bin/vendor" && make install && mv /Users/philip/Code/gulp/images/node_modules/imagemin-jpegoptim/node_modules/jpegoptim-bin/vendor/bin/jpegoptim /Users/philip/Code/gulp/images/node_modules/imagemin-jpegoptim/node_modules/jpegoptim-bin/vendor/jpegoptim
Command failed: /bin/sh -c ./configure --prefix="/Users/philip/Code/gulp/images/node_modules/imagemin-jpegoptim/node_modules/jpegoptim-bin/vendor" --bindir="/Users/philip/Code/gulp/images/node_modules/imagemin-jpegoptim/node_modules/jpegoptim-bin/vendor"

    at ChildProcess.exithandler (child_process.js:751:12)
    at ChildProcess.emit (events.js:110:17)
    at maybeClose (child_process.js:1015:16)
    at Socket.<anonymous> (child_process.js:1183:11)
    at Socket.emit (events.js:107:17)
    at Pipe.close (net.js:485:12)
```

**Comment by Turqueso** [comment]
same error has @philipbordallo. tested on mac os 10.9.5.
tested on Windows 8.1 and I got no error on install, just on mac.

**Comment by smashmiek** [comment]
I had the same issue, and after trying to manually execute the binary, I noticed that it was having issues loading a jpeg library from the system. Reinstalling the jpeg library with brew worked:

http://stackoverflow.com/questions/22994915/fix-gnuplot-installation

**Comment by philipbordallo** [comment]
Yep! `brew install jpeg` fixes it.

Looks like by default OS X has its jpeg library located here: `/System/Library/Frameworks/ImageIO.framework/Resources/libJPEG.dylib`

While jpegoptim is trying to find it here: `/usr/local/lib/libjpeg.8.dylib`

So for OS X at least, there should be a prerequisite of `brew install jpeg`.

**Comment by shinnn** [comment]
I found a `REQUIREMENT` section in the jpegoptim docs. https://github.com/tjko/jpegoptim/blob/b7de23aa4949b7aab62fac6b0f9a52647d3e270e/README#L5-L7

> Independent JPEG Group's jpeg library (libjpeg) version 6b or later

It's a documentation issue. We should update the READMEs of this project and [jpegoptim-bin](https://github.com/imagemin/jpegoptim-bin).

**Comment by philipbordallo** [comment]
@shinnn I can help out with that.

**Comment by shinnn** [comment]
@philipbordallo Thanks.

1. First, add a link to https://github.com/tjko/jpegoptim#readme in [the `Install` section of the jpegoptim-bin doc](https://github.com/imagemin/jpegoptim-bin/tree/5d340dc1283c64405cfd71bb2644e0a6c30eeac1#install).
  * `elm-bin` is a good reference. https://github.com/kevva/elm-bin#install
2. Add a link to https://github.com/imagemin/jpegoptim-bin#install in [the `Install` section of the imagemin-jpegoptim doc](https://github.com/imagemin/imagemin-jpegoptim#install).

**Comment by philipbordallo** [comment]
@shinnn in the [Install section of jpegoptim-bin](https://github.com/imagemin/jpegoptim-bin/tree/5d340dc1283c64405cfd71bb2644e0a6c30eeac1#install) should there at least be references to `brew install libjpeg` for OS X and `apt-get install libjpeg62 libjpeg62-dev` for Linux as there really isn't anything explicit about how to get libjpeg in the [jpegoptim readme](https://github.com/tjko/jpegoptim#readme).

**Comment by shinnn** [comment]
@philipbordallo Yes, jpegoptim readme does not explain enough, but I don't want jpegoptim-bin to have extra information. It should be fixed in the jpegoptim repo first.

Please open a new issue or pull request to the jpegoptim repo to add more detailed requirements. I'll consider adding the references to `brew` and `apt-get` commands to the imagemin-related projects only after jpegoptim author doesn't accept the requirement update.

**Comment by philipbordallo** [comment]
@shinnn I understand, on it.

**Comment by shinnn** [comment]
Thanks, @philipbordallo :)
